metricbeat/docs/fields.asciidoc

```diff
@@ -1075,87 +1072,87 @@ type: long
 
 format: bytes
 
-Data size of couch documents associated with a node (bytes)
+Data size of Couch docs associated with a node (bytes).
 
 
 [float]
 === couchbase.node.couch.spatial.data_size.bytes
 
 type: long
 
-couchSpatialDataSize field
+Size of object data for spatial views (bytes).
 
 
 [float]
 === couchbase.node.couch.spatial.disk_size.bytes
```

Comment:
Question for reviewers: I wonder why this is not called `couchbase.node.couch.spatial.actual_disk_size.bytes` to be consistent with the names of other fields (like `couchbase.node.couch.views.actual_disk_size.bytes`).